action.yml

```diff
@@ -7,6 +7,8 @@ branding:
   icon: alert-circle
   color: gray-dark
 inputs:
+  assignee:
```

Comment:
I wonder if this should be `assignees`? It'd be consistent with the API parameter at least.

Reply:
Sounds good. Made the change 👍